src/Pages/Login/Login.js

```diff
@@ -23,34 +49,37 @@ export default function Login(props) {
     }
   }
 
-  const fields = [
-    {
-      type: 'email',
-      placeholder: 'Email',
-      handleChange: e => setEmail(e.target.value)
-    },
-    {
-      type: 'password',
-      placeholder: 'Password',
-      handleChange: e => setPassword(e.target.value)
+  function getSemesterImage() {
+    const month = new Date().getMonth() + 1;
+    if (month >= 6) {
+      setSemesterImage({
+        ...semesterImage,
+        backgroundImage: `url(${fall})`
+      });
     }
-  ];
+  }
+
+  useEffect(() => {
+    getSemesterImage();
+  });
 
   return (
-    <form onSubmit={handleSubmit}>
-      <div id='body'>
-        <img id='img' alt='sce logo' src='images/SCE-glow.png' />
-        {errorMsg && <span>{errorMsg}</span>}
-        {fields.map((field, index) => {
-          return <LoginInput key={index} field={field} />;
-        })}
-        <button type='submit' id='loginBtn'>
-          Login
-        </button>
-        <p id='SignUp'>
-          <Link to='/register'>Create an account</Link>
-        </p>
-      </div>
-    </form>
+    <Container fluid style={semesterImage}>
```

Comment:
i like the idea of dynamically rendering the style. However the only thing that changes is `backgroundImage`.

Can you:
1. Move all of the non-changing css to a `.css` file?
1. Do sum like
```jsx
<Container fluid class="your-classname-here" style={{backgroundImage: getSemesterImage()}}>
```
3. Then, change your `getSemesterImage` function to just return the proper image like
```js
function getSemesterImage() {
  if(something) {
    return `url(${fall})`
  }  else {
    return /*the other thing*/
  }
}
```